cgi/partial/sbcr_form.tt

```diff
@@ -0,0 +1,341 @@
+  <a style="position:absolute;left:-999px;" href="#" accesskey="n"
+     onfocus="document.getElementById('NoteTextField').focus()">.</a>
+</div>
+<div id="SubmitForm">
+  <table>
+    <tr>
+      <td><input class="review" type="submit" name="submit" value="Submit" accesskey="s"/></td>
+      <td style="padding-left:2em;"><input class="review" type="submit" name="submit"
+          value="Cancel" accesskey="c" onclick="this.form.onsubmit=null;"/></td>
+    </tr>
+  </table>
+  </div>
+  <table>
+  [% IF expert %]
+    <tr>
+      [% checked = (u_swiss || ((status == 2 || status == 3) && data.project.SwissByDefault())) %]
+      <td style="text-align:left;">
+        <input type="checkbox" id="swiss" name="swiss" [% (checked)? 'checked="checked"':'' %]/>
+      </td>
+      <td><label for="swiss">Do not invalidate other reviews<br/>for this volume</label></td>
+    </tr>
+  [% END %]
+  [% holds = crms.CountHolds() %]
+  [% IF !hold %]
+    [% hold = crms.HoldForItem(htid, user) %]
+  [% END %]
+    <tr>
+      <td style="text-align:left;">
+        <input type="checkbox" id="hold" name="hold"
+               [% (holds >=5 && !hold)? 'disabled="disabled"':'' %]
+               [% (hold)? 'checked="checked"':'' %]
+               onclick="toggleVisibility('expiry');"/>
+      </td>
+      <td><label for="hold">Hold for Question</label></td>
+    </tr>
+    <tr>
+      <td colspan="2">
+        <span class="smallishText" [% (holds >=5)? 'style="color:red;"':'' %]>
+            <!--Held reviews will remain unprocessed.-->
+            You currently have [% holds %] out of the maximum 5 volumes on hold.
+            <!--It is up to you to get in touch with an expert.-->
+        </span>
+      </td>
+    </tr>
+  </table>
+  [% IF expert && importUser %]
+    [% reviews = data.reviews %]
+    [% IF reviews.keys.size %]
+      <table>
+        <tr>
+          <td>
+            <div style="line-height:20px;">
+              <strong>Import user review:</strong>
+            </div>
+          </td>
+          <td>
+            <img id="importLoader" src="[% crms.WebPath('web', 'ajax-loader.gif') %]"
+                 alt="loading..." style="display:none;"/>
+          </td>
+        [% FOREACH user IN reviews.keys.sort %]
+          <tr>
+            <td><label for="pull[% user %]">[% user %] ([% reviews.$user.attr %]/[% reviews.$user.reason %])</label></td>
+            <td><input type="radio" name="pullrights" id="pull[% user %]"
+                       [% IF importUser == user %]checked="checked"[% END %]
+                       onclick="insertUserReviewData('[% user %]');"/>
+            </td>
+          </tr>
+        [% END %]
+      </table>
+    [% END %]
+  [% END %]
+</div>
+</form>
+
+<script>
+var gReviewData = [% data.json %];
+addEvent(window, 'load', partial_sbcr_form_mainWindowLoad);
+
+function partial_sbcr_form_mainWindowLoad(e) {
+  [% IF importUser %]
```

Comment:
Could we have `importUser` as a JSON block as well, and then just do something like this (the syntax is probably wrong but you get the idea); then all the JS could be in a static file

```javascript
var importUser = [% importUserJson %];

if(importUser != null) {
  insertUserReviewData(importUser);
}
```